src/dnstable_lookup.c

```diff
@@ -32,6 +32,7 @@
 static bool g_json = false;
 static bool g_Json = false;
 static bool g_add_raw;
+static bool g_summary = false;
```

Comment:
Consider renaming this to `g_count` or similar. We may wish to add a full summary along the lines of `dnsdbq -V summarize` in the future.

Reply:
Done.